source/utils/commit.js

```diff
@@ -1,35 +1,61 @@
-import generatePrompt from "./openai.js";
+import generatePrompt from './openai.js';
 import {execa} from 'execa';
-import readline from "readline";
+import readline from 'readline';
+import React from 'react';
+import {Box, render, Text, useApp} from 'ink';
+import SelectInput from 'ink-select-input';
 
 async function askForCommitMessage() {
-  const prompt = await generatePrompt();
+	const prompt = await generatePrompt();
 
-  const rl = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout,
-  });
+	const rl = readline.createInterface({
+		input: process.stdin,
+		output: process.stdout,
+	});
 
-  if (prompt) {
-    rl.question(`Suggested commit message: ${prompt}\nDo you want to proceed? (y/N) `, (answer) => {
-      if (answer.toLowerCase() === "y") {
-        execa("git", ["commit", "-m", prompt])
-          .then(() => {
-            console.log("Changes committed successfully!");
-          })
-          .catch((error) => {
-            console.error("Failed to commit changes:", error);
-          });
-      } else {
-        console.log("Changes not committed.");
-      }
+	const Demo = () => {
+		const {exit} = useApp();
+		const handleSelect = item => {
+			if (item.value) {
+				execa('git', ['commit', '-m', prompt])
+					.then(() => {
+						console.log('Changes committed successfully!');
+					})
+					.catch(error => {
+						console.error('Failed to commit changes:', error);
+					});
+				
+			} else {
+				console.log('Changes not committed.');
+			}
+      exit();
+		};
 
-      rl.close();
-    });
-  } else {
-    console.log("No changes to commit...");
-    rl.close();
-  }
+		const items = [
+			{
+				label: 'Yes',
+				value: true,
+			},
+			{
+				label: 'No',
+				value: false,
+			},
+		];
```

Comment:
Can we please switch them? No is the default because we want to ensure users don't mindlessly hit enter. The default would be No. When I tried, Yes was highlighted. Thanks @spc-28

Reply:
Hey @spc-28 you forgot to update this, just switch their place. Let's make the No in the top level.

Reply:
@warengonzaga Here i made the default arrow to point No keeping the Yes above No, if you want i can switch their placement.

Reply:
Tested it, and it works!